evdi-kmod-common.spec

```diff
@@ -0,0 +1,94 @@
+Name:          evdi-kmod-common
+Version:       %{_version}
+Release:	1%{?dist}
+Summary:       Userland library for Extensible Virtual Display Interface Kernel module
+URL:           https://github.com/DisplayLink/evdi
+Source0:       %{url}/archive/v%{version}.tar.gz
+Source1:        displaylink.service
+Source2:        99-displaylink.rules
+Source3:        displaylink-sleep-extractor.sh
+# From http://www.displaylink.com/downloads/ubuntu.php
+Source4:	DisplayLink USB Graphics Software for Ubuntu %{_daemon_version}.zip
```

Comment:
You mean so there should be three spec files: evdi, evdi-kmod and displaylink? That would work, but I'd need to move the DKMS part of displaylink.spec into a submodule to avoid the obvious clash.

Reply:
RPM Fusion do not support dkms method, but you can support it if you like to (it would be a separate spec file).
We don't rely on your SCM at all if the spec are imported into RPM Fusion, it will have it's separate life.
You can have all spec into a "rpm" sub-directory of your project.